app/components/entry/ExcerptInput/index.tsx

```diff
@@ -83,6 +87,55 @@ function ExcerptInput<N extends string>(props: Props<N>) {
             </div>
         );
     }
+    if (entryType === 'ATTACHMENT') {
+        const attachmentSrc = imageRaw ?? entryAttachment?.filePreview?.url;
```

Comment:
we shouldn't use imageRaw here, only canonicalImagePreview url

Reply:
Thinking about not using canonical image if we need to pass entryAttachment in every excerptinput anyway.